Library/Formula/zeal.rb

```diff
@@ -0,0 +1,43 @@
+class Zeal < Formula
+  desc "Zeal is a simple offline documentation browser inspired by Dash."
+  homepage "http://zealdocs.org/"
+  url "https://github.com/zealdocs/zeal/archive/v0.1.1.tar.gz"
+  sha256 "f4e959f9bc66a6e350ee8a33d34695379633432d103db9776c0c7a76cbc5a9d6"
+  head "https://github.com/zealdocs/zeal.git"
+
+  depends_on "qt5"
+  depends_on "libarchive"
+
+  patch :DATA
```

Comment:
Can you report this upstream if it hasn't been already and link to that bug report? Thanks!